x/staging/posts/types/polls_test.go

```diff
@@ -15,7 +13,8 @@ import (
 
 func TestPollAnswer_Validate(t *testing.T) {
 	answer := types.NewProvidedAnswer("0", "")
-	require.Equal(t, "answer text must be specified and cannot be empty", answer.Validate().Error())
+	require.Equal(t, "answer text must be specified and cannot be empty: invalid poll",
+		answer.Validate().Error())
```

Comment:
Better to simplify this all together
```suggestion
	require.Err(t, answer.Validate())
```